cpp/tests/io/csv_test.cpp

```diff
@@ -2175,7 +2175,7 @@ TEST_F(CsvReaderTest, DtypesMap)
   auto result = cudf_io::read_csv(in_opts);
 
   const auto result_table = result.tbl->view();
-  assert(result_table->num_columns() == 2);
+  assert(result_table.num_columns() == 2);
```

Comment:
huh. Pretty sure these should not be debug asserts. Never noticed.

Reply:
You want me to just change them to `ASSERT_EQ`? I see some of these in that file.

Reply:
There's five `assert`s in this file. If you could replace them with correct macros, that would be great.